src/ActivityStickyQueues/NonStickyActivities.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Temporalio.Activities;
+
+namespace TemporalioSamples.ActivityStickyQueues;
+
+public class NonStickyActivities
+{
+    private readonly string uniqueWorkerTaskQueue;
+
+    public NonStickyActivities(string uniqueWorkerTaskQueue)
+    {
+        this.uniqueWorkerTaskQueue = uniqueWorkerTaskQueue;
+    }
+
+    [Activity]
+    public Task<string> GetUniqueTaskQueueAsync()
+    {
+        return Task.FromResult(uniqueWorkerTaskQueue);
+    }
```

Comment:
```suggestion
    public NonStickyActivities(string uniqueWorkerTaskQueue) =>
        this.uniqueWorkerTaskQueue = uniqueWorkerTaskQueue;

    [Activity]
    public string GetUniqueTaskQueue() => uniqueWorkerTaskQueue;
```

Not required, just may be easier to read (if .NET gives you guff about getters, you can disable that warning via pragma)